### PR TITLE
Initial changes needed to create a community other than CZO

### PR DESCRIPTION
**Pull request by alvacouch**
The goal here is to strip out the CZO features for communities other than the CZO. This includes: 
1. Enhancing the `access_community`, `access_group`, and `access_resource` management commands for new features. 
2. Removing hard-coded features from the community templates and replacing them with soft-coded features. A lot of things need to disappear for non-CZO communities, including
    * The title builder. 
    * The subject builder. 
    * The CZO banner on the landing page. 
 3. Stripping out unused group-community-group sharing features that are undesirable for BuPuSa and otherwise unused. 
 4. Changing tests to test change (3) completely. 
  
 This is a rough implementation intended to get Clara working on building the community files. I need to follow this with a series of management commands to build her initial community, and then she has to populate them. It does not -- for example -- put up the BuPuSa banner as yet. 

### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [x] Automated Testing

- [x] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]
If you browse to a community other than CZO, CZO customizations disappear.

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|
| NEW DUPLICATED LINES DENSITY | 0.0 |
| NEW SECURITY HOTSPOTS | 0 |
| NEW CODE SMELLS | 0 |
| NEW BUGS | 1 |
| NEW VULNERABILITIES | 0 |

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4330](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4330)

**Review by sblack-usu — APPROVED** [review]
Looks to be a straightforward change to allow other communities to display.

**Comment by alvacouch** [comment]
@sblack-usu I found a bug in the code and pushed a fix. Please re-review. This version is final. I need this to run in prod..... Clara is waiting for it....

**Comment by alvacouch** [comment]
@sblack-usu What this version contains -- that you haven't seen before -- is that I stripped out group-community-group sharing in order to enable the BuPuSa community to work properly.

**Comment by alvacouch** [comment]
(I pushed this code to beta as well for the demo on Monday morning.)

**Comment by alvacouch** [comment]
@sblack-usu Fixed a bunch of dumb things. Should pass tests now.

**Comment by alvacouch** [comment]
@sblack-usu It passed testing. Please review.

File: hs_access_control/models/community.py [hunk 1]
```diff
@@ -184,8 +184,4 @@ def get_resources_with_explicit_access(self, user, group, privilege):
         elif privilege == PrivilegeCodes.CHANGE:  # requires superuser
             return BaseResource.objects.none()
         else:  # VIEW is requested for regular user via community
-            return BaseResource.objects.filter(
-                # The only reasonable protection is VIEW; don't check protection.
-                Q(r2grp__group=group,
-                  r2grp__group__g2gcp__community=self,
-                  r2grp__group__g2gcp__community__c2gcp__group__g2ugp__user=user)).distinct()
+            return BaseResource.objects.none()
```
Comment (sblack-usu) [inline hs_access_control/models/community.py:187]: This can be combined with the CHANGE privilege code.

**Review by sblack-usu — APPROVED** [review]

**Comment by alvacouch** [comment]
@sblack-usu I'll look at the code for CHANGE again, but I'm in a bit of a situation here with BuPuSa.

**Pull request merged**